docs/unumpy_using.rst

```diff
@@ -0,0 +1,35 @@
+.. currentmodule:: unumpy
+
+Using ``unumpy``
+================
+
+``unumpy`` is an in-progress mirror of the NumPy API which allows the user
+to dynamically switch out the backend that they are using. It also allows
+auto-selection of the backend based on the arguments passed into a function.
+
+Currently, only the following functions are supported:
+
+* NumPy `universal functions <https://www.numpy.org/devdocs/reference/ufuncs.html>`_.
+
+  * `ufunc reductions <https://www.numpy.org/devdocs/reference/generated/numpy.ufunc.reduce.html#numpy.ufunc.reduce>`_
+
+* ufunc-based reductions
+
+  * `sum <https://docs.scipy.org/doc/numpy/reference/generated/numpy.sum.html>`_
+  * `prod <https://docs.scipy.org/doc/numpy/reference/generated/numpy.prod.html>`_
+  * `min <https://docs.scipy.org/doc/numpy/reference/generated/numpy.amin.html>`_
+  * `max <https://docs.scipy.org/doc/numpy/reference/generated/numpy.amax.html>`_
+  * `any <https://docs.scipy.org/doc/numpy/reference/generated/numpy.any.html>`_
+  * `all <https://docs.scipy.org/doc/numpy/reference/generated/numpy.all.html>`_
+
+The following backends are supported:
```

Comment:
Can you comment here on whether all listed objects above are supported for all backends? (for example I'm not sure if PyTorch has an exact `ufunc` equivalent).